doc/classes/Input.xml

```diff
@@ -195,6 +195,13 @@
 				By default, the deadzone is automatically calculated from the average of the action deadzones. However, you can override the deadzone to be whatever you want (on the range of 0 to 1).
 			</description>
 		</method>
+		<method name="get_virtual_controller">
+			<return type="VirtualController" />
+			<description>
+				Return platform-specific virtual controller or null. (See [VirtualController])
```

Comment:
Not sure it's necessary to specify the VirtualController since it's present in the method's signature.
```suggestion
				Return the platform-specific virtual controller or [code]null[/code] if it is not available. See also [VirtualController].
```